src/main/scala/it/pagopa/interop/agreementprocess/server/impl/Dependencies.scala

```diff
@@ -80,7 +98,12 @@ trait Dependencies {
     )
     .toFuture
 
-  def fileManager(blockingEc: ExecutionContextExecutor): FileManager = new S3ManagerImpl(blockingEc)
+  def fileManager(blockingEc: ExecutionContextExecutor): FileManager =
```

Comment:
isn't this already checked in commons? 🤔

Reply:
In the commons lib, I see only `FileManager.get` and nothing else.
Am I missing something?

Reply:
no ok, you're right, the helper in commons need the ADT, it doesn't read it from string 👍